### ISSUE TITLE
kingfisher-main: Is postgres.main state file needed?

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/deploy#447]
Hi @RobHooper, when creating the new server, we left in https://github.com/open-contracting/deploy/blob/main/salt/postgres/main.sls in the top file.

However, if these states aren't needed in a non-replication scenario, we can omit it from the top file (and manually remove the cron job it creates).

**Comment by RobHooper** [comment]
I think we can remove this file completely and update references to it in the docs.
We used this custom archive directory prior to configuring pgBackRest but since then, in my opinion; replication should use pgBackRest.
This state is only needed if we wanted to configure replication without backups.

We stopped using this directory with this commit:
https://github.com/open-contracting/deploy/pull/254/files#diff-75192eb288b5986eb9137c193756e8cef4ebeba9e2d88fd010652341f22e6231L131
